README.rst

```diff
@@ -18,7 +18,7 @@ extending the traditional cryptological narrative of "Alice and Bob" by introduc
 .. _OpenSSL: https://www.openssl.org/
 
 
-**Encapsulation**
+**Encryption**
```

Comment:
Not to be confused, with Encapsulation.

Reply:
Yeah, basically the whole "Encapsulation/Decapsulation" concepts should be only internal to pyUmbral. I think this is clearer now, since the PRE module doesn't expose publicly encapsulation/decapsulation functions anymore (these changes are also included in this PR)

Reply:
I'm not sure I fully agree with the assertion that "PRE module doesn't expose publicly encapsulation/decapsulation functions".  To my way of thinking, all of the methods on `Capsule` are "encapsulation/decapsulation functions".  I don't think it makes sense to have a public object called a `Capsule` without also treating its narrative as a public one.